lib/manageiq/api/common/application_controller_mixins/request_body_validation.rb

```diff
@@ -8,6 +8,7 @@ class BodyParseError < ::RuntimeError
 
           def self.included(other)
             ActionController::Parameters.action_on_unpermitted_parameters = :raise
+            other.include(OpenapiEnabled)
```

Comment:
Super minor but can you add a blank line above this...otherwise it looks like this line is related to the previous line.